tests/translator/input/api_with_resource_policy.yaml

```diff
@@ -9,4 +9,19 @@ Resources:
               Action: 'execute-api:Invoke', 
               Resource: ['execute-api:/*/*/*']
             }
+  ImplicitApiFunction:
```

Comment:
Thank you for adding functions to these tests, it helps to have an API definition generated in the tests. We missed that last open api review.